code_challenges/code_wars/highest_scoring_word.py

```diff
@@ -0,0 +1,43 @@
+# https://www.codewars.com/kata/57eb8fcdf670e99d9b000272/train/python
+
+
+def highest_word(test_string):
+    letter_values = {
+        "a": 1,
+        "b": 2,
+        "c": 3,
+        "d": 4,
+        "e": 5,
+        "f": 6,
+        "g": 7,
+        "h": 8,
+        "i": 9,
+        "j": 10,
+        "k": 11,
+        "l": 12,
+        "m": 13,
+        "n": 14,
+        "o": 15,
+        "p": 16,
+        "q": 17,
+        "r": 18,
+        "s": 19,
+        "t": 20,
+        "u": 21,
+        "v": 22,
+        "w": 23,
+        "x": 24,
+        "y": 25,
+        "z": 26,
+    }
+    word_list = test_string.split(" ")
+    highest_word = ""
+    highest_score = 0
+    for word in word_list:
+        word_score = 0
+        for letter in word:
+            word_score = word_score + int(letter_values[letter])
```

Comment:
No need to `int` letter_values[letter] it's already an int

Reply:
I fixed the int() and went ahead and just went for it with the pytest.mark.parametrize. Everything passed. The pytest vscode extension actually sees the parametrize decorator and includes it in the tests. And get this! It breaks it up even better!! Before it pulls in ONE single test that would have all the asserts lumped together as one test so it's just pass or fail (you can see the "test_max_and_min" test just shows up as one test in the screenshot). Now with the parametrize decorator, I can see each one individually independent from each other! Really cool!

![Screenshot from 2023-01-12 19-44-47](https://user-images.githubusercontent.com/28361002/212232550-224b9071-3938-48fc-8336-dce5ef4f931c.png)